hw1/01-selectors.md

```diff
@@ -2,6 +2,33 @@
 
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple#
+5. #fancy pickle
+6. .small
+7. orange.small
+8. bento orange.smallpla
+9. plate, bento
+10. *
+11. plate *
+12. plate + apple
+13. bento ~ pickle
+14. plate > apple
+15. orange:first-child
+16. plate apple,pickle:only-child
```

Comment:
I believe the only way to select multiple elements with the same pseudo-class is to use separate selectors, such as `plate apple:only-child, plate pickle:only-child`, but you could also solve the level without using `apple` or `pickle` at all.